x-parser.js

```diff
@@ -519,15 +532,15 @@ export class XParser {
     const { parsed, notParsed } = XParser.#getErrorInfo(strings, stringsIndex, string, stringIndex);
     const valueForbidden = XParser.#forbiddenTransition(string, stringIndex, value);
     const valueMalformed = XParser.#invalidTransition(string, stringIndex, value);
-    const errorMessagesKey = valueForbidden
-      ? XParser.#valueForbiddenToErrorMessagesKey.get(valueForbidden)
+    const errorMessageKey = valueForbidden
```

Comment:
Refactored some of the variable names to match the new setup. Unfortunately, that made this diff a little extra ugly.